src/style.css

```
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 300;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 900;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  display: block;
+  margin-top: 20px;
+  width: 100%;
+}
+
+.bars {
+  font-family: "Avenir", sans-serif;
```

Move `font-family` to `html` selector
For input use inherit value
```suggestion
  font-family: inherit;
```